# HG changeset patch
# User adf88@interia.pl
# Date 1503166425 -7200
#      Sat Aug 19 20:13:45 2017 +0200
# Branch trunk
# Node ID 2dbfab11f29856ee097312b1ca970190ff692cd6
# Parent  584e1cc2ffb25cb095e7ffa701d23530d5d10ea5
imported patch fix-increase_servicing_interval_tooltip.diff

diff -r 584e1cc2ffb2 -r 2dbfab11f298 src/vehicle_gui.cpp
--- a/src/vehicle_gui.cpp	Sun Aug 13 18:38:42 2017 +0000
+++ b/src/vehicle_gui.cpp	Sat Aug 19 20:13:45 2017 +0200
@@ -1830,7 +1830,7 @@
 		NWidget(WWT_PUSHARROWBTN, COLOUR_GREY, WID_VD_DECREASE_SERVICING_INTERVAL), SetFill(0, 1),
 				SetDataTip(AWV_DECREASE, STR_VEHICLE_DETAILS_DECREASE_SERVICING_INTERVAL_TOOLTIP),
 		NWidget(WWT_PUSHARROWBTN, COLOUR_GREY, WID_VD_INCREASE_SERVICING_INTERVAL), SetFill(0, 1),
-				SetDataTip(AWV_INCREASE, STR_VEHICLE_DETAILS_DECREASE_SERVICING_INTERVAL_TOOLTIP),
+				SetDataTip(AWV_INCREASE, STR_VEHICLE_DETAILS_INCREASE_SERVICING_INTERVAL_TOOLTIP),
 		NWidget(WWT_DROPDOWN, COLOUR_GREY, WID_VD_SERVICE_INTERVAL_DROPDOWN), SetFill(0, 1),
 				SetDataTip(STR_EMPTY, STR_SERVICE_INTERVAL_DROPDOWN_TOOLTIP),
 		NWidget(WWT_PANEL, COLOUR_GREY, WID_VD_SERVICING_INTERVAL), SetFill(1, 1), SetResize(1, 0), EndContainer(),
